pkg/common/telemetry/names.go

```diff
@@ -271,6 +271,9 @@ const (
 	// ExpiresAt tags registration entry expiration
 	ExpiresAt = "expires_at"
 
+	// Hint tags entry hint
```

Comment:
```suggestion
	// Hint tags registration entry hint
```

Reply:
Thank you for the review, @MarcosDY! [401a07e](https://github.com/spiffe/spire/pull/3926/commits/401a07edcec1ce80d2c600507906ebab6bb26001)